sdk/cosmosdb/cosmos/src/queryExecutionContext/Aggregators/AverageAggregator.ts

```diff
@@ -36,7 +33,7 @@ export class AverageAggregator implements Aggregator {
    * @instance
    */
   public getResult() {
-    if (this.sum == null || this.count <= 0) {
+    if (!this.sum || !this.count || this.count <= 0) {
```

Comment:
i think there are differences in behavior here between undefined and null

Reply:
I'm about to push work on the sum aggregator and can likely address all of these

Reply:
> I'm about to push work on the sum aggregator and can likely address all of these

@zfoster by all means, feel free to close this and take those files off the exclusion list in your PR.